### PR TITLE
🎨 Beautify Buy credits

### PR DESCRIPTION
**Pull request by odeimaiz**
## What do these changes do?

This PR prettifies the One Time Payment UI and it also redesigns the Auto Recharge one by updating the form fields and the UI.

One Time payment:
image

Auto recharge:
image

### CODE REVIEW

**Review by pcrespov — COMMENTED** [review]
I would prefer to check with PR https://github.com/ITISFoundation/osparc-simcore/pull/4946 before you merge this one. 
For instance 
- the minimum threshold to top-up is defined in the backend and returned read-only in the auto-recharge info. ( you hard-coded 25 in the help)
- extra help on the monthly limit  (perhaps an info button that shows more details or a link to a manual??)

**Review by elisabettai — APPROVED** [review]

**Review by matusdrobuliak66 — APPROVED** [review]
I would maybe add temporary to auto recharge some notification "coming soon" while it is not yet implemented.

**Review by ignapas — APPROVED** [review]
👍🏼

**Review by sanderegg — APPROVED** [review]

File: services/static-webserver/client/source/class/osparc/desktop/credits/AutoRecharge.js [hunk 1]
```diff
@@ -250,7 +263,7 @@ qx.Class.define("osparc.desktop.credits.AutoRecharge", {
         maxWidth: 200,
         center: true
       });
-      const successfulMsg = this.tr("Auto recharge was successfully enabled");
+      const successfulMsg = this.tr("Auto recharge was successfully enabled. Coming soon.");
```
Comment (jsaq007) [inline services/static-webserver/client/source/class/osparc/desktop/credits/AutoRecharge.js:266]: Does this mean auto recharge isn't available yet?

**Review by jsaq007 — COMMENTED** [review]
:+1:

File: services/static-webserver/client/source/class/osparc/desktop/credits/AutoRecharge.js (hunk 1 above)
Comment (odeimaiz) [reply services/static-webserver/client/source/class/osparc/desktop/credits/AutoRecharge.js:266]: Correct.

**Pull request merged**